src/WordPress/Blueprints/Resources/Resolver/ResourceResolverCollection.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace WordPress\Blueprints\Resources\Resolver;
+
+use WordPress\Blueprints\Model\DataClass\ResourceDefinitionInterface;
+
+class ResourceResolverCollection implements ResourceResolverInterface
+{
+
+    /** @var ResourceResolverInterface[] */
+    protected array $ResourceResolvers;
+
+    public function __construct(
+        $ResourceResolvers
+    ) {
+        $this->ResourceResolvers = $ResourceResolvers;
+    }
+
+    public static function getResourceClass(): string
+    {
+        throw new \RuntimeException('Not implemented');
+    }
+
+    public function parseUrl(string $url): ?ResourceDefinitionInterface
+    {
+        foreach ($this->ResourceResolvers as $handler) {
+            $resourceFilePath = $handler->parseUrl($url);
+            if ($resourceFilePath) {
+                return $resourceFilePath;
+            }
+        }
+
+        return null;
+    }
+
+    public function supports(ResourceDefinitionInterface $resourceDefinition): bool
```

Comment:
In here, `$resources` was renamed to `$resourceDefinition` which is fine but inconsistent with the rename in the `parseUrl` method. Let's choose one name and stick to it. I picked `$resource` because it's nice and short.

Reply:
Reverted.